dkms.conf

```diff
@@ -6,14 +6,19 @@
 PACKAGE_NAME="xpmem"
 PACKAGE_VERSION="2.7.3"
 
+# Need to check if the configure script exists and run autogen.sh if it doesn't.
+# This has to be in a standalone script because DKMS does not support shell
+# commands in PRE_BUILD.
```

Comment:
maybe add `or passing any argument`

Reply:
Done, though I used `nor` instead of `or`